tfhe/src/integer/server_key/radix_parallel/tests_cases_unsigned.rs

```diff
@@ -2623,6 +2623,21 @@ where
 
     executor.setup(&cks, sks.clone());
 
+    // Test case of division by 0
+    // This is mainly to show we know the behaviour of division by 0
+    // using the current algorithm
+    for clear_0 in [0, rng.gen::<u64>() % modulus] {
+        let ctxt_0 = cks.encrypt(clear_0);
+        let ctxt_1 = cks.encrypt(0u64);
+
+        let (q_res, r_res) = sks.div_rem_parallelized(&ctxt_0, &ctxt_1);
```

Comment:
should we have the checked_div somewhere as well ?

Also is this test case shared with signed integers ?

Reply:
Signed integer already have their own case for this

Checked div is tested in the docstrings

Reply:
thx